README.md

```diff
@@ -1 +1,180 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Weboack <a name="about-project"></a>
+
+**Leaderboard** list app is a project to act as a score board for a computer game.
+
+## 🛠 Built With <a name="built-with"></a>
+- Webpack
+- Html
+- Css
+- Javascript
+- Gitflow
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Markup</summary>
+  <ul>
+    <li><a href="https://html5.org/">HTML5</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Interaction</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Easy navigation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need an IDE such as VS code, sublime, atom e.t.c:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+ $ cd my-folder
+ $ git@github.com:Cisco-the-wayword/Leaderboard.git
+```
+
+### Install
+
+Install git to work on this project with:
+
+```sh
+   $ git config --global user.name "First Last"
+
+   $ git config --global user.email "example@example.com"
+```
+
```

Comment:
- Hey, I think the command you want to add is `npm install` to install all the dependencies needed to run the app. Kindly update this. if you want your users to install git, kindly add the link to git documentation as the code you've added won't set up git correctly. We want to add correct information for our users. Please, kindly update this to make your readme more professional.